src/erc721/ERC721.sol

```diff
@@ -0,0 +1,237 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity >=0.8.0;
+
+/// @notice Modern and gas efficient ERC-721 + ERC-20/EIP-2612-like implementation.
+contract ERC721 {
+    /*///////////////////////////////////////////////////////////////
```

Comment:
these documentation lines 🤢

Reply:
do you mean these guys?
`/*///////////////////////////////////////////////////////////////` 

I blame transmissions :~0 But have grown on me

Reply:
I overuse them a tad in solmate might strip em out except for the more complex contracts 😅